x/ibc/02-client/keeper/client.go

```diff
@@ -72,13 +86,18 @@ func (k Keeper) UpdateClient(ctx sdk.Context, clientID string, header exported.H
 	}
 
 	var (
-		consensusState exported.ConsensusState
-		err            error
+		consensusState      exported.ConsensusState
+		err                 error
+		oldHeaderNextValSet *tmtypes.ValidatorSet // TODO: should I get this from the msg or internal query?
 	)
 
 	switch clientType {
 	case exported.Tendermint:
-		clientState, consensusState, err = tendermint.CheckValidityAndUpdateState(clientState, header, ctx.ChainID())
+		unbondingTime := k.stakingKeeper.UnbondingTime(ctx)
+		trustingPeriod := (2 / 3) * unbondingTime
```

Comment:
q: should the `2/3` be an IBC parameter?

Reply:
I think this is something we should have the users pass in via the `TrustOptions` struct to initialize the lite client.

Reply:
This should be a parameter, see this PR - https://github.com/cosmos/ics/pull/367